content/en/docs/installation/helm.md

```diff
@@ -0,0 +1,174 @@
+> providers such as Let's Encrypt.
+
+## Installing with Helm
+
+> **Note**: cert-manager should never be embedded as a sub-chart into other Helm charts.
+> cert-manager manages non-namespaced resources in your cluster and should only be installed once.
+
+### Prerequisites
+
+- Helm v3 installed
+
+### Steps
+
+In order to install the Helm chart, you must follow these steps:
+
+#### 1. Add the Jetstack Helm repository:
+
+> **Warning**: It is important that this repository is used to install
+> cert-manager. The version residing in the [Helm stable repository](https://github.com/helm/charts/tree/master/stable/cert-manager) is
+> *deprecated* and should *NOT* be used.
+
+```bash
+$ helm repo add jetstack https://charts.jetstack.io
+```
+
+#### 2. Update your local Helm chart repository cache:
+
+```bash
+$ helm repo update
+```
+
+#### 3. Install `CustomResourceDefinitions`
+
+cert-manager requires a number of CRD resources to be installed into your
+cluster as part of installation.
+
+This can either be done manually, using `kubectl`, or using the `installCRDs`
+option when installing the Helm chart.
+
+> **Note**: If you're using a `helm` version based on Kubernetes `v1.18` or below (Helm `v3.2`) `installCRDs` will not work with cert-manager `v0.16`.
+> For more info see the [v0.16 upgrade notes](../upgrading/upgrading-0.15-0.16/#helm)
+
+##### Option 1: installing CRDs with `kubectl`
+
+Install the `CustomResourceDefinition` resources using `kubectl`:
+
+```bash
+$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.4.0/cert-manager.crds.yaml
+```
+
+##### Option 2: install CRDs as part of the Helm release
+
+To automatically install and manage the CRDs as part of your Helm release, you
+must add the `--set installCRDs=true` flag to your Helm installation command.
+
+Uncomment the relevant line in the next steps to enable this.
+
+#### 3. Install cert-manager
+
+To install the cert-manager Helm chart, use the [Helm install command](https://helm.sh/docs/helm/helm_install/) as described below.
+
+```bash
+$ helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.4.0 \
+  # --set installCRDs=true
+```
+
+On the [cert-manager's ArtifactHub page](https://artifacthub.io/packages/helm/cert-manager/cert-manager) a full list of the available **Helm values** can be found.
+
+The example below shows how to tune the cert-manager installation by overwriting the default Helm values:
+
+```bash
+$ helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.4.0 \
+  --set prometheus.enabled=false \  # Example: disabling prometheus using a Helm parameter
+  --set webhook.timeoutSeconds=4s   # Example: changing the wehbook timeout using a Helm parameter
+```
+
+Once you have deployed cert-manager, you can verify the installation
+[here](../verify/).
```

Comment:
suggestion: make the verb the link, rather than adding a "here"

```suggestion
Once you have deployed cert-manager, you can [verify](../verify/) the installation.
```